uploader.json

```diff
@@ -3365,6 +3365,11 @@
             ],
             "symbolic_name": "setup_named_chroot"
         },
+        {
+            "file": "/usr/lib/udev/rules.d/59-fc-wwpn-id.rules",
+            "pattern": [],
+            "symbolic_name": "udev_fc_wwpn_id_rules"
+        },
```

Comment:
@huali027 please move this spec after `sap_host_profile` so that at least the last few specs are in alpha sorted order by symbolic name.